### DISCUSSION TITLE
Pole figure showing misorientations to a given orientation

### DISCUSSION THREAD

**Discussion by vitesh13** [mtex-toolbox/mtex#1619] [Ask Anything]
Hello,

Is there any way to show on a pole figure, all the orientations that lie within certain misorientation tolerance from a given orientation? 
An illustration can be taken as the figure that I attach. 

Lets say I have an orientation represented by "Teal/green marker" in the image and I somehow get all the misorientations within 5 degrees. And lets say they lie within the area marked by red dashed line. 
![Pole_figure_example](https://user-images.githubusercontent.com/41839982/198633995-3d523590-ace1-4f84-8333-8a032d71d9df.png)

I think something like this should be possible, however, I am not able to find any idea for it.

**Comment by kilir** [comment]
Hi,
what is shown in a polefigue are not orientations but directions.

If you want to find directions within 5 degree from a ref erence direction `dir(angle(dir, ref_dir)<5*degree)` should be an way to do that.

Cheers,
Rüdiger
